quickstarts/microsoft.search/azure-search-create-private-endpoint/azuredeploy.json

```diff
@@ -0,0 +1,200 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "minLength": 2,
+            "maxLength": 60,
+            "type": "string",
+            "metadata": {
+                "description": "Service name must only contain lowercase letters, digits or dashes, cannot use dash as the first two or last one characters, cannot contain consecutive dashes, and is limited between 2 and 60 characters in length."
+            }
+        },
+        "sku": {
+            "defaultValue": "standard",
+            "allowedValues": [
+                "basic",
+                "standard",
+                "standard2",
+                "standard3",
+                "storage_optimized_l1",
+                "storage_optimized_l2"
+            ],
+            "type": "string",
+            "metadata": {
+                "description": "The pricing tier of the search service you want to create (for example, basic or standard)."
+            }
+        },
+        "replicaCount": {
+            "defaultValue": 1,
+            "minValue": 1,
+            "maxValue": 12,
+            "type": "int",
+            "metadata": {
+                "description": "Replicas distribute search workloads across the service. You need at least two replicas to support high availability of query workloads (not applicable to the free tier)."
+            }
+        },
+        "partitionCount": {
+            "defaultValue": 1,
+            "allowedValues": [
+                1,
+                2,
+                3,
+                4,
+                6,
+                12
+            ],
+            "type": "int",
+            "metadata": {
+                "description": "Partitions allow for scaling of document count as well as faster indexing by sharding your index over multiple search units."
```

Comment:
A search unit is defined as replica * partition so this phrasing is potentially confusing.  What do you think about: "Partitions are units of storage. Increasing partitions adds capacity and speed by spreading an index over multiple shards."